Source/WebCore/css/CSSPropertyInitialValues.cpp

```diff
@@ -0,0 +1,617 @@
+    case CSSPropertyPrintColorAdjust:
+        return CSSValueEconomy;
+    case CSSPropertyScrollTimelineAxis:
+    case CSSPropertyViewTimelineAxis:
+        return CSSValueBlock;
+    case CSSPropertyScrollTimelineName:
+    case CSSPropertyViewTimelineName:
+        return CSSValueNone;
+    case CSSPropertyViewTimelineInset:
+        return CSSValueAuto;
+    case CSSPropertyStrokeColor:
+        return CSSValueTransparent;
+    case CSSPropertyStrokeLinecap:
+        return CSSValueButt;
+    case CSSPropertyStrokeLinejoin:
+        return CSSValueMiter;
+    case CSSPropertyStrokeMiterlimit:
+        return InitialNumericValue { 4, CSSUnitType::CSS_NUMBER };
+    case CSSPropertyStrokeWidth:
+        return InitialNumericValue { 1, CSSUnitType::CSS_PX };
+    case CSSPropertyTabSize:
+        return InitialNumericValue { 8, CSSUnitType::CSS_NUMBER };
+    case CSSPropertyTextAlign:
+        return CSSValueStart;
+    case CSSPropertyTextDecorationStyle:
+        return CSSValueSolid;
+    case CSSPropertyTextBoxEdge:
+        return CSSValueAuto;
+    case CSSPropertyTextOrientation:
+        return CSSValueMixed;
+    case CSSPropertyTextOverflow:
+        return CSSValueClip;
+    case CSSPropertyTextWrapMode:
+        return CSSValueWrap;
+    case CSSPropertyTextWrapStyle:
+        return CSSValueAuto;
+    case CSSPropertyTransformBox:
+        return CSSValueViewBox;
+    case CSSPropertyTransformStyle:
+        return CSSValueFlat;
+    case CSSPropertyTransitionProperty:
+        return CSSValueAll;
+    case CSSPropertyWritingMode:
+        return CSSValueHorizontalTb;
+    case CSSPropertyTextSpacingTrim:
+        return CSSValueSpaceAll;
+    case CSSPropertyTextAutospace:
+        return CSSValueNoAutospace;
+    case CSSPropertyWhiteSpaceCollapse:
+        return CSSValueCollapse;
+    case CSSPropertyFieldSizing:
+        return CSSValueFixed;
+    default:
+        RELEASE_ASSERT_NOT_REACHED();
+    }
+}
+
+static bool isValueIDPair(const CSSValue& value, CSSValueID valueID)
+{
+    return value.isPair() && isValueID(value.first(), valueID) && isValueID(value.second(), valueID);
+}
+
+static bool isNumber(const CSSPrimitiveValue& value, double number, CSSUnitType type)
+{
+    return value.primitiveType() == type && !value.isCalculated() && value.valueNoConversionDataRequired<double>() == number;
+}
+
+static bool isNumber(const CSSPrimitiveValue* value, double number, CSSUnitType type)
+{
+    return value && isNumber(*value, number, type);
+}
+
+static bool isNumber(const CSSValue& value, double number, CSSUnitType type)
+{
+    return isNumber(dynamicDowncast<CSSPrimitiveValue>(value), number, type);
+}
+
+static bool isNumber(const RectBase& quad, double number, CSSUnitType type)
+{
+    return isNumber(quad.protectedTop(), number, type)
+        && isNumber(quad.protectedRight(), number, type)
+        && isNumber(quad.protectedBottom(), number, type)
+        && isNumber(quad.protectedLeft(), number, type);
```

Comment:
We use `protectedTop` here but just `top` in `isValueID`. Not that we need to fix this, just observing this inconsistency.